cocos/2d/CCFontFNT.h

```diff
@@ -136,7 +140,10 @@ class CC_DLL FontFNT : public Font
     
 public:
     
-    static FontFNT * create(const std::string& fntFilePath, const Vec2& imageOffset = Vec2::ZERO);
```

Comment:
It will break compatibility.

Reply:
Yep, I mentioned that in the previous discussion, but I'll re-add it as a wrapper around the new functionality. I'll have it done soon.

Reply:
@minggo Would it be ok if I added the CC_DEPRECATED_ATTRIBUTE to the functions with the old API?

Reply:
Yep, it is better.

Reply:
Ok, it's been committed.  If any automated tests fail I'll sort them out pretty quickly, otherwise it should be all good.  Let me know if there's anything else I should address for this PR.

Reply:
Thanks, i need take some time to review it.